e2e/components/Slider/Slider-test.avt.e2e.js

```diff
@@ -92,6 +92,46 @@ test.describe('@avt Slider', () => {
     await expect(page).toHaveNoACViolations('Slider-with-layer');
   });
 
+  test.slow(
+    '@avt-advanced-states slider with custom format',
```

Comment:
Hey @dkaushik95 
The `@avt-advanced-states` and `@avt-keyboard-nav` should be separate tests.

In the  `@avt-advanced-states` for the new stories, just keep this line:
```js
      await expect(page).toHaveNoACViolations('Slider-with-custom-value-label');
```

For the `@avt-keyboard-nav` you can keep only he keyboard navigation you are testing.

Here it is an example of the difference between the two tests:

![Screenshot 2024-06-27 at 14 08 28](https://github.com/carbon-design-system/carbon/assets/52183462/ed255a02-8d30-4336-9b14-6985166d4ae6)

Reply:
Got it! Thanks for the info! I'm new to the structure :D I'll spin out a commit soon